src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -103,3 +93,321 @@
     </div>
   {% endfor %}
 {% endmacro %}
+
+
+{#----------------------------------- Supporting Macros ----------------------------------------------#}
+
+{% macro _claimRejectedForClaimant(claim) %}
+  {% set defendantName = claim.claimData.defendant.name %}
```

Comment:
is it possible to set defendantName and claimantName at top level (which you already did) and use it every where within macros instead of setting multiple times within macros? Not sure if it possible but that will save couple of lines of code.

Reply:
just wondering if there is any benefit in splitting caseHistory.njk file into different files(if possible)? it seems too long now

Reply:
Im just seeing these comments now.
I wanted it to be set at the top of the file but it wasnt being called as i wanted it to be.
Im currently working on splitting all the files now so there is a pretty large restructure thats going to get pushed soon. 
You're 100% right, its too long and now (after my next push) it will be a lot more manageable